qiskit/chemistry/applications/molecular_ground_state_energy.py

```diff
@@ -29,6 +29,8 @@
 class MolecularGroundStateEnergy:
     """ Molecular ground state energy chemistry application """
 
+    #TODO this needs to be deprecated in view of the new Ground State interface
```

Comment:
We need to fix this TODO and properly deprecate `MolecularGroundStateEnergy`

Reply:
this is done